git_machete/docs.py

```diff
@@ -140,6 +142,26 @@
           <b>-b, --branch=<branch></b>      Branch to set the annotation for.
           <b>-H, --sync-github-prs</b>      Annotate with GitHub PR numbers and authors where applicable.
     """,
+    "clean": """
+        <b>Usage:
+          git machete clean [-c|--checkout-my-github-prs]
+
+        If invoked without any flag, deletes untracked and unmanaged branches.
+
+        If invoked with ``-c`` or ``--checkout-my-github-prs``, checkouts your open PRs into local branches.
```

Comment:
Ban on using `checkout` as a verb, other than in infinitive (which I also frown upon, but kind of okay) :whip: